src/main/scala/com/advancedtelematic/director/db/DeviceUpdateAssignmentRepository.scala

```diff
@@ -54,14 +52,15 @@ protected class DeviceUpdateAssignmentRepository()(implicit db: Database, ec: Ex
     namespace: Namespace,
     deviceId: DeviceId,
     correlationId: Option[CorrelationId],
-    updateId: Option[UpdateId])
+    updateId: Option[UpdateId])(fileCacheRepository: FileCacheRepository)
   : DBIO[Int] = for {
     version <- fetchLatestAction(namespace, deviceId).asTry.flatMap {
       case Success(x) => DBIO.successful(x)
       case Failure(NoTargetsScheduled) => DBIO.successful(0)
       case Failure(ex) => DBIO.failed(ex)
     }
-    newVersion = version + 1
+    latestTargetsVersion <- fileCacheRepository.fetchLatestVersionAction(deviceId)
```

Comment:
It's named `currentTimestampVersion` in the other changes, is it intended?

Reply:
I'll change it to be consistent, but director just generates all roles at the same time so the version for timestamps and targets is the same

Reply:
Ah thanks, didn't know that 👍